Build failure: emulationstation-de

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

The error can be reproduced by launching a shell that installs the package from nixpkgs unstable:
```bash
$ NIXPKGS_ALLOW_INSECURE=1 nix-shell -I nixpkgs=channel:nixos-unstable -p emulationstation-de
```



### Can Hydra reproduce this build failure?

Please select the Hydra Status.

### Link to Hydra build job

https://hydra.nixos.org/search?query=emulationstation-de

### Relevant log output

```console
building '/nix/store/rjxcm8736dskb8kl5mc0p4z25bj0003i-emulationstation-de-3.2.0.drv'...
Running phase: unpackPhase
unpacking source archive /nix/store/820lahvckr3ggrjgqnfm0w5nb1682l64-source
source root is source
Running phase: patchPhase
applying patch /nix/store/5aq6x78l0c8v7vd7qgf2lvb9j8ihipdv-001-add-nixpkgs-retroarch-cores.patch
patching file resources/systems/linux/es_find_rules.xml
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/share/doc/es-de -DCMAKE_INSTALL_INFODIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/include -DCMAKE_INSTALL_SBINDIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/strip -DCMAKE_RANLIB=/nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/ranlib -DCMAKE_AR=/nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0 -DAPPLICATION_UPDATER:BOOL=FALSE
-- The C compiler identification is GNU 14.2.1
-- The CXX compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at /nix/store/ad4c8nx2lkjs4zq0sj9jdpn7w45v9ccs-cmake-3.31.6/share/cmake-3.31/Modules/FindPackageHandleStandardArgs.cmake:233 (message):
  Could NOT find OpenGL (missing: OPENGL_opengl_LIBRARY OPENGL_glx_LIBRARY
  OPENGL_INCLUDE_DIR)
Call Stack (most recent call first):
  /nix/store/ad4c8nx2lkjs4zq0sj9jdpn7w45v9ccs-cmake-3.31.6/share/cmake-3.31/Modules/FindPackageHandleStandardArgs.cmake:603 (_FPHSA_FAILURE_MESSAGE)
  /nix/store/ad4c8nx2lkjs4zq0sj9jdpn7w45v9ccs-cmake-3.31.6/share/cmake-3.31/Modules/FindOpenGL.cmake:579 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  CMakeLists.txt:130 (find_package)


-- Configuring incomplete, errors occurred!
error: builder for '/nix/store/rjxcm8736dskb8kl5mc0p4z25bj0003i-emulationstation-de-3.2.0.drv' failed with exit code 1;
       last 25 log lines:
       > Running phase: configurePhase
       > fixing cmake files...
       > cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/share/doc/es-de -DCMAKE_INSTALL_INFODIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/include -DCMAKE_INSTALL_SBINDIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/strip -DCMAKE_RANLIB=/nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/ranlib -DCMAKE_AR=/nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/mibraz8b952brm2yh36lj6hfkyl1ij0p-emulationstation-de-3.2.0 -DAPPLICATION_UPDATER:BOOL=FALSE
       > -- The C compiler identification is GNU 14.2.1
       > -- The CXX compiler identification is GNU 14.2.1
       > -- Detecting C compiler ABI info
       > -- Detecting C compiler ABI info - done
       > -- Check for working C compiler: /nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/gcc - skipped
       > -- Detecting C compile features
       > -- Detecting C compile features - done
       > -- Detecting CXX compiler ABI info
       > -- Detecting CXX compiler ABI info - done
       > -- Check for working CXX compiler: /nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/g++ - skipped
       > -- Detecting CXX compile features
       > -- Detecting CXX compile features - done
       > CMake Error at /nix/store/ad4c8nx2lkjs4zq0sj9jdpn7w45v9ccs-cmake-3.31.6/share/cmake-3.31/Modules/FindPackageHandleStandardArgs.cmake:233 (message):
       >   Could NOT find OpenGL (missing: OPENGL_opengl_LIBRARY OPENGL_glx_LIBRARY
       >   OPENGL_INCLUDE_DIR)
       > Call Stack (most recent call first):
       >   /nix/store/ad4c8nx2lkjs4zq0sj9jdpn7w45v9ccs-cmake-3.31.6/share/cmake-3.31/Modules/FindPackageHandleStandardArgs.cmake:603 (_FPHSA_FAILURE_MESSAGE)
       >   /nix/store/ad4c8nx2lkjs4zq0sj9jdpn7w45v9ccs-cmake-3.31.6/share/cmake-3.31/Modules/FindOpenGL.cmake:579 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
       >   CMakeLists.txt:130 (find_package)
       >
       > 
       > -- Configuring incomplete, errors occurred!
       For full logs, run 'nix log /nix/store/rjxcm8736dskb8kl5mc0p4z25bj0003i-emulationstation-de-3.2.0.drv'.
```

### Additional context

Issue #380330 also concerns a build failure for this package, but if I recall correctly, that issue was resolved after #397909 was merged. Additionally, Hydra displays a 500 error when searching for this package.

This is my first open issue on nixpkgs, if there is any missing information or I have made any mistakes please let me know

### System metadata

- system: `"x86_64-linux"`
 - host os: `Linux 6.14.4-cachyos, NixOS, 24.11 (Vicuna), 24.11.20250428.330d0a4`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.14`
 - nixpkgs: `/nix/store/xcyzpa1jx529h82p01bfhkgwp4v5m7a6-source`


### Notify maintainers

@ivarmedi 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/em/emulationstation-de/package.nix
+++ b/pkgs/by-name/em/emulationstation-de/package.nix
@@ -17,6 +17,7 @@
   poppler,
   pugixml,
   SDL2,
+  libGL,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
@@ -57,6 +58,7 @@ stdenv.mkDerivation (finalAttrs: {
     poppler
     pugixml
     SDL2
+    libGL
   ];
 
   cmakeFlags = [ (lib.cmakeBool "APPLICATION_UPDATER" false) ];

